package.json

```diff
@@ -8,6 +8,7 @@
   "private": true,
   "scripts": {
     "configure-mapbox": "scripts/setup-mapbox-sdk-walkthrough.sh",
+    "setupNewDotWebForEmulators": "scripts/setup-newdot-web-emulators.sh",
```

Comment:
Are these supposed to be a part of prebuild? or atleast added to the README? @hayata-suenaga ?

Reply:
I think README.md will be updated here
https://github.com/Expensify/App/pull/28422

Reply:
@shubham1206agra can you check the current README change in [that PR](https://github.com/Expensify/App/pull/28422/files) and put a comment in that PR on additional instruction we need to add because of these new scripts if there are any?